core/Azure.Mcp.Core/src/Commands/CommandFactory.cs

```diff
@@ -299,7 +299,7 @@ public string RemoveRootGroupFromCommandName(string fullCommandName)
     }
 
     /// <summary>
-    /// Gets the service area given the full command name (i.e. 'storage_account_list' or 'azmcp_storage_account_list' would return 'storage').
+    /// Gets the service area given the full command name (i.e. 'storage_account_list' or 'azmcp_storage_account_get' would return 'storage').
```

Comment:
It would be hard to keep track of the tool name changes in comments. Can we make the comment describe how to discover the service area for a command?